### PR TITLE
core-js ->zloirock/core-js

### PR DESCRIPTION
**Pull request by Aaike**

### CODE REVIEW

File: package-overrides/github/fronteed/iCheck@1.0.2.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "main": "icheck.js",
   "registry": "jspm",
   "format": "global",
-  "files":["icheck.js","icheck.min.js","skins/*"],
+  "files":["icheck.js","icheck.min.js","skins/**/*"],
```
Comment (guybedford) [inline package-overrides/github/fronteed/iCheck@1.0.2.json]: Globs in the files object are not currently supported.

File: package-overrides/npm/js-beautify@1.5.5.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "files": ["js/**/*"]
```
Comment (guybedford) [inline package-overrides/npm/js-beautify@1.5.5.json]: Again

File: package-overrides/github/fronteed/iCheck@1.0.2.json (hunk 1 above)
Comment (Aaike) [inline package-overrides/github/fronteed/iCheck@1.0.2.json]: really ? i tested this override it seems to work ?

Comment (guybedford) [inline package-overrides/github/fronteed/iCheck@1.0.2.json]: Oh right! Sorry forgot about that.

File: registry.json [hunk 3]
```diff
@@ -81,6 +81,7 @@
   "backbone.babysitter": "github:marionettejs/backbone.babysitter",
   "backbone.wreqr": "github:marionettejs/backbone.wreqr",
   "baconjs": "npm:baconjs",
+  "js-beautify": "github:beautify-web/js-beautify",
```
Comment (guybedford) [inline registry.json]: alphabetical order

**Comment by Aaike** [comment]
Hi, is there something i should still fix in this PR ?

**Comment by guybedford** [comment]
Looks good. Sorry - updated commits don't send notifications that's all.

**Pull request merged**